src/mmw/js/src/core/utils.js

```diff
@@ -254,6 +266,12 @@ var utils = {
 
             return l < r ? 1 : l > r ? -1 : 0;
         };
+    },
+
+    totalForPointSourceCollection: function(collection, key) {
+        return _.reduce(collection, function(accumulator, element) {
```

Comment:
This may be made easier to read as a combination of `_.sum` and `_.map`:

```js
return _.sum(_.map(collection, function(element) {
  return element.attributes[key] || 0;
}));
```